.travis.yml

```diff
@@ -1,40 +1,98 @@
 sudo: required
+_aliases:
+  - &ui-tests
+    addons:
+      firefox: latest-nightly
+      hosts: example.com
```

Comment:
Can you also update [the docs](https://github.com/mozilla/addons-frontend/blob/master/tests/ui/README.md) for how to do this manually? I assume there is an extra step now where you have to add `example.com 127.0.0.1` to `/etc/hosts`, right?